packages/react-native/template/package.json

```diff
@@ -29,7 +29,7 @@
     "metro-react-native-babel-preset": "0.76.0",
     "prettier": "^2.4.1",
     "react-test-renderer": "18.2.0",
-    "typescript": "4.8.4"
+    "typescript": "5.0.4"
```

Comment:
Should we go for flexible patch or minor here? At least patch?
```suggestion
    "typescript": "~5.0.4"
```

Reply:
minor safe imo ^ presuming future minor vers are backwards-compat
otherwise feel free to revert or go for patch

Reply:
gotcha makes sense, so we'll avoid the `^`